### DISCUSSION TITLE
If I want to query a random piece of data how to write

### DISCUSSION THREAD

**Discussion by Drizzle365** [tortoise/tortoise-orm#1173] [General]
I did not find the corresponding interface of order by rand() in the official documentation, as if the official interface only supports ascending descending order, if I want to randomly get a piece of data to change how to deal with it

**Comment by long2ice** [comment]
See https://github.com/tortoise/tortoise-orm/issues/938

**Reply by Drizzle365** [reply]
Thank you so much
